src/test/java/uk/gov/companieshouse/documentsigningapi/controller/SignDocumentControllerIntegrationTest.java

```diff
@@ -195,15 +197,6 @@ void setUp() {
         setUpSignedDocumentBucket();
     }
 
-    @AfterAll
```

Comment:
If this was here before it implies a context persists between tests and is populated with env vars, worth checking it's ok to remove this

Reply:
The change from System rules to system stubs means that the tear down is not required. System stubs resets them automatically.